<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="drop_widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>30</y>
      <width>831</width>
      <height>841</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>821</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(20, 33, 61);
background-size: cover;
background-position: top;
border-top-left-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>551</width>
       <height>821</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255,255,255);
border-bottom-right-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Answer with yes or no</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hd">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Headache</string>
     </property>
    </widget>
    <widget class="QPushButton" name="d_button">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>720</y>
       <width>255</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 8px;
background:#14213d;
height: 30px;
color: #fff;

}
QPushButton:hover{
background-color:rgb(0,0,0);
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>Diagnose</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#fff;</string>
     </property>
     <property name="text">
      <string>Medical Expert system</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cls_button">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>10</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{

border-radius: 13.5px;
background:#14213d;
color: #fff;

}
QPushButton:hover{
background-color:#e63946;
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>780</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#14213d;</string>
     </property>
     <property name="text">
      <string>Abwape.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="min_butto">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>10</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{

border-radius: 13.5px;
background:#e5e5e5;
color: #000;

}
QPushButton:hover{
background-color:#06d6a0;
color:#fff;
}
QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}

</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bp">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>160</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Back pain</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cp">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Chest pain</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="c">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>240</y>
       <width>255</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Cough</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="f">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>300</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Fainting</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ft">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>340</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Fatigue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="se">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>380</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Sunken eyes</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lbt">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>430</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Low body temperature</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="r">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>470</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Restlessness</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="st">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>510</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Sore throat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fe">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>550</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Fever</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="n">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>600</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Nausea</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bv">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>660</y>
       <width>255</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:none;
border-bottom:2px solid #14213d;
height: 30px;
padding-bottom:7px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Blurred vision</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>271</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#fff;</string>
     </property>
     <property name="text">
      <string>Hi! I am Dr.Abwao, I am here to  make your health better.For that you'll have to answer a few questions about your conditions.Are you experiencing any of the following symptoms</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
